src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/SelectVariants.java

```diff
@@ -489,8 +510,7 @@ public void onTraversalStart() {
 
         // Look at the parameters to decide which analysis to perform
         discordanceOnly = discordanceTrack != null;
-        if (discordanceOnly) {
-            logger.info("Selecting only variants discordant with the track: " + discordanceTrack.getName());
+        if (discordanceOnly) {logger.info("Selecting only variants discordant with the track: " + discordanceTrack.getName());
```

Comment:
Should fix the spacing/indentation here before merging

Reply:
^I agree with david can you revert this please?